src/quo2/screens/selectors/disclaimer.cljs

```diff
@@ -0,0 +1,26 @@
+(ns quo2.screens.selectors.disclaimer
+  (:require [quo.react-native :as rn]
+            [reagent.core :as reagent]
+            [quo2.components.selectors.disclaimer :as quo2]
+            [quo2.foundations.colors :as colors]))
+
+(defn cool-preview []
+  (let [checked? (reagent/atom false)]
```

Comment:
this is an interesting way of managing the checkbox's state.

Checkbox depends on reagent atom <- which we pass in disclaimer component <- which will be passed to disclaimer component from its parent
(and maybe more ...) that's weird

Usually, native components manage their state themselves and we only pass two fields 
1. `is-checked?` for setting default value and
2. `on-change` for any callback related to the new change.

So, I guess it will be better if we move this reagent/atom in the checkbox itself. wdyt?

Reply:
cool, will adjust! thanks for the suggestions! :D

Reply:
hmm, are we sure we don't want the value externally? Having the value passed as a prop allows for easier access to the `value` of the checkbox.
in my experience accessing internal state of a component often leads to the value being "off" by one render.

I will bring this discussion to mobile channel

Reply:
agreed on internal state 👍